test/test_v2/Makefile

```
@@ -1,46 +1,58 @@
-INSTALLDIR = $(HOME)/pmix-install
-INCLUDES = -I. -I../../test -I../../include -I../.. -I../../src/include
-vpath %.h .:../../test:../../include:../..:../../src/include
-LDIR = $(INSTALLDIR)/lib
+# Set the environment variable PMIX_INSTALLDIR to point to your local PMIx install directory
+ifeq ($(strip $(PMIX_INSTALLDIR)),)
+$(info ******** Set PMIX_INSTALLDIR environment variable to point to PMIx install directory ********)
+$(info ******** Defaulting to $(HOME)/pmix-install ********)
+PMIX_INSTALLDIR = $(HOME)/pmix-install
+endif
```

I made a comment [here](https://github.com/openpmix/openpmix/pull/1960#pullrequestreview-557133591) that you might consider changing (so it has a default value, but can be changed via the envar):
```suggestion
endif
PMIX_INSTALLDIR ?= $(HOME)/pmix-install
```

---

Well, the existing code accomplishes the same thing but also warns if the env var isn't set (accomplished through the `ifeq ($(strip $(PMIX_INSTALLDIR))`). I think I'd prefer the original because it warns... what do you think?

---

@jjhursey

---

I changed it to your method because Howard preferred no warning if the variable isn't set.